### PR TITLE
Ritiksoni00 patch 2

### PR DESCRIPTION
**Pull request by ritiksoni00**
waiting for the Revieww! gg

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ Blogger is a basic blogging application which includes user to create account an
 - **Frontend:** HTML/CSS
 - **Backend:** Django
 
-
+asdfadsfda
```
Comment (Kajol-Kumari) [inline README.md:8]: Please remove this unnecessary change @ritiksoni00

File: .github/Feature_request.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+---
```
Comment (Kajol-Kumari) [inline .github/Feature_request.md:1]: Please put this file under `./github/ISSUE_TEMPLATE/` folder

File: .github/Pull_request_Template.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
+# Related Issuse or bug
+  - Info about Issue or bug
```
Comment (Kajol-Kumari) [inline .github/Pull_request_Template.md:2]: Please Add a line here `Fixes: #[issue number that will be closed through this PR]`

Comment (ritiksoni00) [reply .github/Pull_request_Template.md:2]: Added!

File: README.md (hunk 1 above)
Comment (ritiksoni00) [reply README.md]: sorry!

File: .github/Feature_request.md (hunk 2 above)
Comment (ritiksoni00) [reply .github/Feature_request.md:1]: Done!

**Comment by ritiksoni00** [comment]
i made another Pr which is perfect.
